hummingbot/connector/exchange/kucoin/kucoin_api_order_book_data_source.py

```diff
@@ -107,13 +107,12 @@ async def get_ws_connection_context(cls, throttler: Optional[AsyncThrottler] = N
         throttler = throttler or cls._get_throttler_instance()
         async with aiohttp.ClientSession() as session:
             url = CONSTANTS.BASE_PATH_URL + CONSTANTS.PUBLIC_WS_DATA_PATH_URL
-            async with throttler.execute_task(CONSTANTS.PUBLIC_WS_DATA_PATH_URL):
-                async with session.post(url, data=b'') as resp:
-                    response: aiohttp.ClientResponse = resp
-                    if response.status != 200:
-                        raise IOError(f"Error fetching Kucoin websocket connection data."
-                                      f"HTTP status is {response.status}.")
-                    data: Dict[str, Any] = await response.json()
```

Comment:
I think we should always pass through the throttler before performing any request. If there are no limits for the particular endpoint that should be detected by the throttler based on its configuration.

Reply:
Agreed and reverted